lib/tdf3/src/client/DecoratedReadableStream.ts

```diff
@@ -32,8 +32,8 @@ export abstract class DecoratedReadableStream {
   manifest: Manifest;
   upsertResponse?: UpsertResponse;
 
-  constructor(byteLimit: number, underlyingSource: UnderlyingSource) {
-    this.stream = new ReadableStream(underlyingSource, { highWaterMark: byteLimit });
+  constructor(underlyingSource: UnderlyingSource) {
+    this.stream = new ReadableStream(underlyingSource, { highWaterMark: 1 });
```

Comment:
IIUC this will always immediately write whatever to the underlying stream. Which makes sense, as this is just supposed to be a proxy, not a buffer.